test/test_nestedtensor.py

```diff
@@ -6050,6 +6050,49 @@ def test_copy_(self, device):
         ):
             a.copy_(b)
 
+    def test_index_put_(self, device):
```

Comment:
it'd be nice to validate that this works on a non-contiguous with holes NJT as well

Reply:
one of the input is non-contiguous with holes! ADded it to the opinfo